src/main/java/de/czymm/serversigns/parsing/operators/PlaceholderOperator.java

```diff
@@ -0,0 +1,49 @@
+package de.czymm.serversigns.parsing.operators;
+
+import org.bukkit.entity.Player;
+
+import de.czymm.serversigns.ServerSignsPlugin;
+import de.czymm.serversigns.signs.ServerSign;
+import me.clip.placeholderapi.PlaceholderAPI;
+
+public class PlaceholderOperator extends ConditionalOperator {
+
+    public PlaceholderOperator() {
+        super("placeholder", true);
+    }
+
+    @Override
+    public ParameterValidityResponse checkParameterValidity(String params) {
+        boolean isValid = params.indexOf('=') > 0 && params.length() - 2 >= params.indexOf('=');
+        return new ParameterValidityResponse(isValid, "Parameter must be in the format <placeholder>=<match>");
```

Comment:
Can you add support of > and < for number only ?
That we can support comparison between number ?